tests/test_router_bgp.rb

```diff
@@ -192,15 +224,19 @@ def test_routerbgp_set_get_bestpath
       assert(bgp.bestpath_cost_community_ignore, "vrf #{vrf}: "\
              'bgp bestpath_cost_community_ignore should be enabled')
       bgp.bestpath_med_confed = true
-      assert(bgp.bestpath_med_confed, "vrf #{vrf}: "\
-             'bgp bestpath_med_confed should be enabled')
+      if node.client.api == 'NXAPI' ||
```

Comment:
No in this case and a few others they worked in IOS XR at the global level (vrf = default) but not under a vrf. I'll add an appropriate comment to these.